test/libethereum/BlockChainTests.cpp

```diff
@@ -34,10 +34,13 @@ using namespace json_spirit;
 using namespace dev;
 using namespace dev::eth;
 
+
 namespace dev {
 
 namespace test {
 
+eth::Network TestBlockChain::sealEngineNetwork = eth::Network::FrontierTest;
```

Comment:
you say you check that blocknumber in the tests, but why do you need to specify the seal engine for each test?